src/main/python/hydra/rmqtest/runtest.py

```diff
@@ -63,7 +63,9 @@ def rerun_test(self, options):
                                   msg_requested_rate=self.options.msg_rate)
         l.info("PUB server updated")
 
-        self.reset_all_app_stats(self.rmqsub)
+        # Pass signals in groups of apps
```

Comment:
Not sure if I understand. process grouping is same is done at the backend irrespective of rmq, zmq. 
How is cassandra coming into picture ? you have to add unit tests for cassandara once it comes in.
The current implementation is being exercised by rmq test.
Next is to actually start launching brokers, c++ clients.